apps/src/templates/sectionProgress/standards/StandardsIntroDialog.jsx

```diff
@@ -21,17 +23,32 @@ const styles = {
   }
 };
 
+/*
+Dialog that show the first time a teacher goes to the
+Standards view of the Progress Tab in Teacher Dashboard
+ */
+
 class StandardsIntroDialog extends Component {
   static propTypes = {
     isOpen: PropTypes.bool.isRequired,
-    handleConfirm: PropTypes.func.isRequired
+    setCurrentUserHasSeenStandardsReportInfo: PropTypes.func.isRequired
+  };
+
+  dismissStandardsDialog = () => {
+    $.ajax({
+      url: '/dashboardapi/v1/users/me/set_standards_report_info_to_seen',
+      type: 'post',
```

Comment:
when you put "me" in this route it figures who the current user is?

Reply:
That's what it seems to do. I based it off other similar routes